pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -230,7 +230,7 @@ func NewCmdCompletion(fullName string, f *clientcmd.Factory, out io.Writer) *cob
 		cmdHelpName = "openshift"
 	}
 
-	cmd := kcmd.NewCmdCompletion(f, out)
+	cmd := kcmd.NewCmdCompletion(f, out, "\n")
```

Comment:
Initially I went with a space, but that looked bad, so I fall back to newline, which looks much better in the generated file.